jbpm-installer/src/main/resources/build.properties

```diff
@@ -18,57 +18,57 @@ release.version=7.33.0-SNAPSHOT
 #   jBPM.version=${snapshot.version}
 #   jBPM.url=https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.drools&a=droolsjbpm-bpms-distribution&v=${jBPM.version}&e=zip&c=bin
 jBPM.version=${snapshot.version}
-jBPM.url=https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.drools&a=droolsjbpm-bpms-distribution&v=${jBPM.version}&e=zip&c=bin
+jBPM.url=https://origin-repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.drools&a=droolsjbpm-bpms-distribution&v=${jBPM.version}&e=zip&c=bin
```

Comment:
@mbiarnes do we also need to change to _origin_?

Reply:
This is probably to avoid proxy issues which JBoss repository have had lately.